lxd/db/cluster/entities.go

```diff
@@ -16,13 +16,177 @@ import (
 	"github.com/canonical/lxd/shared/entity"
 )
 
-// EntityType is a database representation of an entity type.
+// EntityType is a wrapper for the underlying entityType. Allowing us to implement sql.Scanner and driver.Valuer.
```

Comment:
> Allowing us to implement sql.Scanner and driver.Valuer.

I dont follow this. Where is that implemented? There is no `Scan` function https://pkg.go.dev/database/sql#Scanner

Reply:
There is a scan function already: https://github.com/canonical/lxd/blob/939abdd1afdc5a96bb0c2b4a89b92140103aac4b/lxd/db/cluster/entities.go#L58

This commit updates it for the new type definition: https://github.com/canonical/lxd/pull/13262/commits/7ea324235659d15a52639dfdc6f1b02e024a680e